### PR TITLE
Magiclysm: Demon spiders

### PR DESCRIPTION
**Pull request by KorGgenT**
#### Summary

```SUMMARY: Mods "Magiclysm: Adds demon spiders and a demon spider queen lair"```

NOTE: these monsters will not work properly without #32108

Creates demon spiders, which are terrifying gigantic spiders the size of vehicles. creates the demon spider queen lair as an overmap special, and makes demon spiderlings appear sometimes in forests.

image

image

### CODE REVIEW

File: data/mods/Magiclysm/worldgen/demon_spider_lair.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        "I": "f_desk",
+        "D": "f_desk",
+        "S": "f_sink",
+        "~": "f_bathtub",
+        "m": "f_bigmirror",
+        "b": "f_bookcase",
+        "M": "f_bookcase",
+        "X": "f_magiclysm_translocator_gate",
+        "W": "f_water_heater"
+      }
```
Comment (AMurkin) [inline data/mods/Magiclysm/worldgen/demon_spider_lair.json]:
```suggestion
      },
```

File: data/mods/Magiclysm/terrain.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (AMurkin) [inline data/mods/Magiclysm/terrain.json]:
```suggestion
[
```

File: data/mods/Magiclysm/terrain.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "bash": {
+      "str_min": 30,
+      "str_max": 180,
+      "sound": "crunch!",
+      "sound_fail": "whump!",
+      "ter_set": "t_dirt",
+      "items": [ { "item": "ash", "count": [ 3, 10 ] } ]
+    }
+  }
+}
```
Comment (AMurkin) [inline data/mods/Magiclysm/terrain.json]:
```suggestion
]
```

**Comment by Mecares** [comment]
Nice, i always was terrified by teleporting Blink Spiders, maybe they have a place in magiclysm as well.

**Comment by KorGgenT** [comment]
maybe, but these don't teleport

**Pull request merged**